src/main/java/ladder/domain/Record.java

```diff
@@ -8,22 +8,22 @@
 import java.util.stream.Collectors;
 
 public class Record {
-  private final Map<Player, Integer> values;
+  private final Map<Player, Position> values;
 
-  private Record(Map<Player, Integer> values) {
+  private Record(Map<Player, Position> values) {
     this.values = values;
   }
 
   public static Record generate(final Players players, final Ladder ladder) {
     final List<Player> playersValues = players.getValues();
 
-    Map<Player, Integer> result = playersValues.stream()
-            .collect(Collectors.toMap(Function.identity(), player -> ladder.getLastDepthStartLineIndex(playersValues.indexOf(player))));
+    Map<Player, Position> result = playersValues.stream()
+            .collect(Collectors.toMap(Function.identity(), player -> ladder.getLastDepthStartLinePosition(playersValues.indexOf(player))));
     return new Record(Collections.unmodifiableMap(result));
   }
 
-  public int getPlayerIndexOf(Player player) {
-    return values.getOrDefault(player, 0);
+  public Position getPlayerPosition(Player player) {
+    return values.getOrDefault(player, new Position(0));
   }
```

Comment:
```suggestion
  public Position getPlayerPosition(Player player) {
    return values.getOrDefault(player, new Position());
  }
```

```java
class Position {
    Position () {
        this(0);
    }
}
```

기본 생성자를 Position 클래스에 정의해주시거나, 0 값을 가진 Positon 을 생성하는 static 팩토리 메서드를 만드는 건 어떨까요?
의미를 유추할 수 없는 값을 하드 코딩하는 것은 가능한 피하는 편이 좋습니다.

Reply:
헙 맞네요 감사합니다 !